CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ This file is used to list changes made in each version of the splunk cookbook.
 
 ## Unreleased
 
+- Fixes Issue [#204](https://github.com/sous-chefs/chef-splunk/issues/204)# Please enter the commit message for your changes. Lines starting
```

Comment:
Looks like you didn't mean to add this.
```suggestion
- Fixes Issue [#204](https://github.com/sous-chefs/chef-splunk/issues/204)#
```

Reply:
fixed the change log entry with 4aad88c0fefcb2e41065471d6bf3d8394f16ec38